Tests/AnimationViewTests.swift

```diff
@@ -74,67 +82,87 @@ final class AnimationViewTests: XCTestCase {
       ("automatic", .automatic),
     ]
 
-    let animation = LottieAnimation.named(
+    let animationSupportedByCoreAnimationRenderingEngine = LottieAnimation.named(
       "Issues/issue_1877",
       bundle: .lottie,
       subdirectory: Samples.directoryName)
 
-    XCTAssertNotNil(animation)
-
-    let window = UIWindow()
-
-    for (test, values) in tests.enumerated() {
-      for engine in engineOptions {
-        let animationView = LottieAnimationView(
-          animation: animation,
-          configuration: .init(renderingEngine: engine.engine))
-
-        window.addSubview(animationView)
-        defer {
-          animationView.removeFromSuperview()
-        }
-
-        let animationPlayingExpectation = XCTestExpectation(
-          description: "Animation playing case \(test) on engine: \(engine.label)")
-
-        let animationCompleteExpectation = XCTestExpectation(
-          description: "Finished playing case \(test) on engine: \(engine.label)")
-
-        animationView.play(fromFrame: values.fromFrame, toFrame: values.toFrame, loopMode: .playOnce) { finished in
-          XCTAssertTrue(
-            finished,
-            "Failed case \(test) on engine: \(engine.label)")
-
-          XCTAssertEqual(
-            animationView.currentFrame,
-            values.toFrame,
-            accuracy: 0.01,
-            "Failed case \(test) on engine: \(engine.label)")
-
-          XCTAssertFalse(
-            animationView.isAnimationPlaying,
-            "Failed case \(test) on engine: \(engine.label)")
-
-          animationCompleteExpectation.fulfill()
-        }
-
-        DispatchQueue.main.asyncAfter(deadline: .now() + 0.05) {
-          animationPlayingExpectation.fulfill()
+    let animationUnsupportedByCoreAnimationRenderingEngine = LottieAnimation.named(
```

Comment:
Updated this existing test to include an additional case. This case fails without the change to `LottieAnimationView`.